nifi-extension-bundles/nifi-cql-bundle/nifi-cql-nar/pom.xml

```diff
@@ -0,0 +1,51 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
given 'nifi-cql-bundle' is the top level concept it really should say the word 'cassandra' fully written out.  

so 'nifi-cassandra-bundle' at the top level at least.

Reply:
The bundle includes a driver NAR for ScyllaDB support, so I think calling it `nifi-cql-bundle` makes more sense here. That said, it's been a while AFAICT since Cassandra and ScyllaDB support were discussed, so I'm open to splitting this into two separate bundles because we can't predict where Cassandra and ScyllaDB will be in the future.